geniza/corpus/templates/corpus/document_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static i18n %}
+
+{% comment %}Translators: Title for document search page{% endcomment %}
+{% translate 'Search Documents' as doctitle%}
+
+{% block title %}{{ doctitle }}{% endblock title %}
+
+{% block main %}
+<div class="container search">
+    <h1>{{ doctitle }}</h1>
+
+    <form>
+    {{ form }}
+    {% for field, errors in form.errors.items %}
+    {# no clean way to get field labels here, so we just collapse the error lists #}
+        {% for error in errors %}
+            <li>{{ error }}</li>
+        {% endfor %}
+    {% endfor %}
+    </form>
+
+    {% comment %}Translators: number of search results{% endcomment %}
+    {% blocktranslate count counter=total %}
+        1 result
+    {% plural %}
+        {{ counter }} total results
+    {% endblocktranslate %}
+
+    <ol>
+    {% for document in documents %}
+        <li class="result">
+
+            <dl class="tags">
+                <dt class="sr-only">{% translate 'Tags' %}</dt>
+                {% for tag in document.tags %}
+                <dd class="tag">#{{ tag }}</dd>
+                {% endfor %}
+            </dl>
+
+            <h2><b>{{ document.type }},</b> {{ document.shelfmark|join:" + " }}</h2>
```

Comment:
I think we want the shorter form here that displays a plus or some other indicator for joins, but not every shelfmark